File: Web/KombiCim.Data/Models/Mobile/Dtos/WeatherDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace KombiCim.Data.Models.Mobile.Dtos
{
    public class WeatherDto
    {
        public double Temperature { get; set; }
        public double Humidity { get; set; }
        public DateTime Date { get; set; }
    }
}

File: Android/app/src/main/java/com/kc/kombicim/Model/Dto/WeatherDto.kt
package com.kc.kombicim.Model.Dto

import com.google.gson.annotations.SerializedName

data class WeatherDto (

    @SerializedName("Temperature") val temperature : Double,
    @SerializedName("Humidity") val humidity : Double,
    @SerializedName("Date") val date : String
)